install_config/monitoring/overview.adoc

```diff
@@ -0,0 +1,39 @@
+= Overview
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+{product-title} ships with a pre-configured and self-updating monitoring stack that is based on the link:https://prometheus.io/[Prometheus] open source project and its wider eco-system. It provides monitoring of cluster components and ships with a set of alerts to immediately notify the cluster administrator about any occurring problems and a set of link:https://grafana.com/[Grafana] dashboards.
+
+image::monitoring-diagram.png[]
+
+Highlighted in the diagram above, at the heart of the monitoring stack sits the {product-title} Cluster Monitoring Operator (CMO), which watches over the deployed monitoring components and resources, and ensures that they are always up to date.
+
+The Prometheus Operator (PO) creates, configures, and manages Prometheus and Alertmanager instances. It also automatically generates monitoring target configurations based on familiar Kubernetes label queries.
+
+In addition to Prometheus and Alertmanager, {product-title} Monitoring also includes link:https://github.com/prometheus/node_exporter[node-exporter] and link:https://github.com/kubernetes/kube-state-metrics[kube-state-metrics]. Node-exporter is an agent deployed on every node to collect metrics about it. The kube-state-metrics exporter agent converts Kubernetes objects to metrics consumable by Prometheus.
+
+The targets monitored as part of the cluster monitoring are:
+
+* Prometheus itself
+* Prometheus-Operator
+* cluster-monitoring-operator
+* Alertmanager cluster instances
+* Kubernetes apiserver
+* kubelets (the kubelet embeds cAdvisor for per container metrics)
+* kube-controllers
+* kube-state-metrics
+* node-exporter
+* etcd (if etcd monitoring is enabled)
+
+All these components are automatically updated.
+
+[NOTE]
+====
+In order to be able to deliver updates with guaranteed compatibility, configurability of the {product-title} Monitoring stack is limited to the explicitly available options.
+====
+
+.Additional resources
+
+For more information about the {product-title} Cluster Monitoring Operator, see the link:https://github.com/openshift/cluster-monitoring-operator[Cluster Monitoring Operator] GitHub project.
```

Comment:
No github links.

Reply:
This one, Vikram said, is on a case-by-case basis. Some GitHub links will be allowed. I'm leaving the GitHub links for now, I think they qualify for inclusion. If not, it will come up on the review for 4.0.